app/js/lib/foxbox/settings.js

```diff
@@ -29,29 +29,26 @@ const storage = localStorage ? localStorage : {
   clear: () => {},
 };
 
-export default class Settings extends Model {
+export default class Settings extends EventDispatcher {
   constructor() {
+    super(['session-changed', 'polling-setting-changed']);
```

Comment:
Honestly it feels like session should not belong to ```Settings```, but that's what we have now, and I'm trying to differentiate settings from non-settings with ```-setting-``` part in the event name....